test/jdk/valhalla/valuetypes/PrimitiveTypeConversionTest.java

```diff
@@ -61,8 +61,8 @@ static Value narrow(Value.ref v) {
     @Test
     public static void primitiveWidening() throws Throwable {
         MethodHandles.Lookup lookup = MethodHandles.lookup();
-        MethodHandle mh1 = lookup.findStatic(PrimitiveTypeConversionTest.class, "narrow", methodType(Value.class, Value.ref.class));
-        MethodHandle mh2 = mh1.asType(methodType(Value.class, Value.class));
+        MethodHandle mh1 = lookup.findStatic(PrimitiveTypeConversionTest.class, "narrow", methodType(Value.class.asValueType(), Value.ref.class));
```

Comment:
```suggestion
        MethodHandle mh1 = lookup.findStatic(PrimitiveTypeConversionTest.class, "narrow", methodType(Value.class.asValueType(), Value.class.asPrimaryType()));
```

This change will be needed anyway if it's decided to disallow `Value.ref.class` and `Value.val.class`.  We can wait until that discussion is resolved.